coordinator/Services/JudgerService.cs

```diff
@@ -44,5 +47,46 @@ public class JudgerService {
                 .SingleOrDefaultAsync();
             return judger;
         }
+
+        public async Task<List<JudgerEntry>> QueryJudger(
```

Comment:
Naming overly general: should include something like `tag`.

Reply:
This is the only (and the most general) query method on judger available for now. If one day we need to add another (which is unlikely) we'll change the name by then.